### PR TITLE
Update jakarta test

### PR DESCRIPTION
**Pull request by MonicaisHer**

### CODE REVIEW

File: data/latest/test-device-virtual.sh [hunk 1]
```diff
@@ -62,7 +62,7 @@ while true; do
         print_error_logs
         echo "invalid JSON response from core-metadata"
         exit 1
-    elif [ "$(edgexfoundry.curl -s localhost:59881/api/v2/device/all | $JQ 'map(select(.name == "Random-Boolean-Device")) | length')" -lt 1 ]; then
+    elif [ "$(edgexfoundry.curl -s localhost:59881/api/v2/device/all | $JQ '[.devices[] | select(.name == "Random-Boolean-Device")] | length')" -lt 1 ]; then
```
Comment (farshidtz) [inline data/latest/test-device-virtual.sh]: Please revert or drop this commit and add this change in another PR.

**Review by farshidtz — CHANGES_REQUESTED** [review]
Thanks for the updates.

File: data/latest/test-device-virtual.sh (hunk 1 above)
Comment (farshidtz) [reply data/latest/test-device-virtual.sh]: It goes beyond the scope of jakarta updates.

Comment (MonicaisHer) [reply data/latest/test-device-virtual.sh]: Thanks! Now it has been split to: https://github.com/canonical/edgex-checkbox-provider/pull/33

**Review by farshidtz — APPROVED** [review]

**Pull request merged**